lib/MicroServiceBusHost.js

```diff
@@ -1475,6 +1475,7 @@ function MicroServiceBusHost(settingsHelper) {
     MicroServiceBusHost.prototype.Start = function (testFlag) {
         try {
             util.displayAnsiiLogo();
+            log(`Core version: ${this.version()}`);
```

Comment:
This will not help as `var pjson = require('../package.json')` is somehow cached and will return previous version on update.
Scenario:
1. Node is running 3.2
2. New version of core is released (3.3)
3. Node is restarted, and downloads 3.3
4. After restart: `require('../package.json').version` => 3.2
5. Node is restarted again
6. `require('../package.json').version` => 3.3

I'm not sure why this is... Perhaps we can try running `npm cache clean`